nab/detectors/skyline/skyline_detector.py

```diff
@@ -11,6 +11,13 @@
 
 
 class SkylineDetector(AnomalyDetector):
+  """ Detects anomalies using Etsy Skyline's ensemble of algorithms from
+  https://github.com/etsy/skyline/blob/master/src/analyzer/algorithms.py.
+  Each algorithm in the ensemble gives a binary vote for the data record.
+  The original implementation used a majority voting scheme to classify a record
+  as anomalous. Here we improve the detector's performance by taking
+  average over all the votes to get an anomaly score.
```

Comment:
"... performance by using the average of the algorithms' votes as an anomaly score."